api/src/modules/suppliers/suppliers.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, PrimaryGeneratedColumn, Column, BaseEntity } from 'typeorm';
+import { entityStatus } from 'utils/entity-status.enum';
+
+@Entity()
+export class Suppliers extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ type: 'ltree', nullable: false, unique: true })
+  path: string;
+
+  @Column({ nullable: true })
```

Comment:
are we sure we want the name to be nullable?

Reply:
@tiagojsag  Right.  Fix -> fda24d1bbf475b29231af2b03860ac10fca012cb

Besides that, just followed the provided definition:
![image](https://user-images.githubusercontent.com/37807221/122936437-730e1500-d371-11eb-85d1-5e052318f8d0.png)
So I guess if its not marked as NOT NULL, then can be nullable

TypeORM,s default value for nullable is `false`, but just added it for better readability IMO

Reply:
we shouldn't take the model definition by Francis/Elene without adding our own input and experience to it. I'm sure they put a lot of work and care into it, but I'm sure they are open to our input on it. If we anticipate any problem with a particular definition, we can talk with them about it or, for minor things like this, we can just go with our suggestion, which can easily be modified in the future if needed anyways.